content/desktop/release-notes.md

```diff
@@ -24,11 +24,57 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.22.0
+
+{{< release-date date="2023-08-03" >}}
+
+{{< desktop-install all=true version="4.22.0" build_path="/117440/" >}}
+
+### Upgrades
+- [Compose v2.21.0](https://github.com/docker/compose/releases/tag/v2.21.0)
+- [Docker Scout CLI v0.24.1](https://github.com/docker/scout-cli/releases/tag/v0.24.1). With Docker Scout, you can now:
+  - Docker Desktop can now manage temporary and cached files with `docker scout cache`.
+  - Docker Desktop can now manage environments with `docker scout environment`.
+  - You can now configure default organization with `docker scout config`.
+  - List packages of an image with their vulnerabilities with `docker scout cves --format only-packages`.
+  - You can enroll an organization with Docker scout with `docker scout enroll`.
+  - Docker Desktop allow sto analyse and compare local file system with `docker scout cves --type fs`.
```

Comment:
These switch between "Docker Desktop can now..." and "You can now...", I think they should all be made consistent

Reply:
I thought it was so repetitive, but I left it with `you can now`